rustler/src/types/binary.rs

```diff
@@ -132,18 +121,15 @@ pub struct Binary<'a> {
 impl<'a> Binary<'a> {
     /// Releases a given `OwnedBinary` to the VM.
     /// After this point the binary will be immutable.
-    pub fn from_owned(mut bin: OwnedBinary, env: Env<'a>) -> Self {
-        bin.release = false;
+    pub fn from_owned(bin: OwnedBinary, env: Env<'a>) -> Self {
+        let mut bin = std::mem::ManuallyDrop::new(bin);
```

Comment:
It might be a good idea to add a comment here explaining that we intentionally skip the `drop()` at the end of scope.

Reply:
good suggestion, will add.